modules/server-compatibility/pages/server-compatibility-xdcr.adoc

```diff
@@ -87,14 +91,44 @@ It pushes data from the primary data center to the secondary data centers, where
 image::ROOT:{image-xdcr-data-replication-to-read-only}[,std-image-size]
 ====
 
+=== Bi-directional Active-Active XDCR
+
+Sync Gateway supports bi-directional replication using XDCR between two active mobile clusters.
+This enables active-active deployments where both clusters can process writes simultaneously.
+
+TIP: Verify that you installed Couchbase Server 7.6 or later and configured your Sync Gateway nodes with `shared_bucket_access=true` and `import_docs=true`
```

Comment:
The requirement is 7.6.5 and above for active-active XDCR.

Without XDCR, any patch release for 7.6 is fine for Sync Gateway

As above, shared_bucket_access=false will not allow a database to start up, so I don't know if it needs to be mentioned here as well.